### PR TITLE
Ember 3.10.1

### PR DESCRIPTION
**Pull request by stopfstedt**
consolidates various 3.10.0 updates.

### CODE REVIEW

**Comment by stopfstedt** [comment]
closes #671 #673  #674

**Comment by stopfstedt** [comment]
breaks on form validation library, see ember-cp-validations, issue 640.

File: .eslintrc.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 module.exports = {
   root: true,
+  parserOptions: {
```
Comment (jrjohnson) [inline .eslintrc.js]: All of these config options come from `@ilios/eslint-config-ember-addon` and ndon't need to be specified here.

File: package.json [hunk 2]
```diff
@@ -90,26 +90,26 @@
...
-    "qunit-dom": "^0.8.0",
+    "qunit-dom": "^0.8.4",
     "sass": "^1.13.4",
     "stylelint": "^10.0.1",
     "stylelint-config-recommended-scss": "^3.2.0",
     "stylelint-scss": "^3.5.1"
   },
   "engines": {
-    "node": ">= 10.*"
+    "node": "8.* || >= 10.*"
```
Comment (jrjohnson) [inline package.json]: We dropped node 8 support in #428

Comment (stopfstedt) [reply package.json]: handled.

File: .eslintrc.js (hunk 1 above)
Comment (stopfstedt) [reply .eslintrc.js]: removed.

**Pull request merged**